### ISSUE TITLE
AWSAuthUIViewController iOS 13 modalPresentationStyle = UIModalPresentationFullScreen;

### ISSUE THREAD

**Issue by BillBunting** [aws-amplify/aws-sdk-ios#1963]
**Describe the bug**
iOS 13 changed the default modalPresentationStyle. The AWSAuthUIViewController for the sign-in screen should be full screen. 

**Which AWS service(s) are affected?**
AWSAuthUI

**Expected behavior**
The AWSMobileClient.default().showSignIn should present full screen. 

**Screenshots**
![IMG_2209](https://user-images.githubusercontent.com/5861783/66517155-9b2be680-eab0-11e9-9063-d9733370a48e.jpeg)

**Environment(please complete the following information):**
 - SDK Version: 2.11.1
 - Dependency Manager: Cocoapods, Carthage
 - Swift Version : 5.0

**Device Information (please complete the following information):**
 - Device: Any
 - iOS Version: iOS 13
 - Specific to simulators: No

**Additional context**

Code suggested to make change: Add `navController.modalPresentationStyle = UIModalPresentationFullScreen; as example shown below.` in two locations where the `navController` is presented. 

```
// new signIn screen which allows optional federation
+ (void)presentViewControllerWithConfig:(NSMutableDictionary<NSString *, id> *)configDictionary
                                    completionHandler:(AWSAuthUIExtendedCompletionHandler)completionHandler {
    
    UINavigationController *navigationController = (UINavigationController *)configDictionary[@"navigationController"];
    AWSAuthUIConfiguration *config = [AWSAuthUIViewController getDefaultAuthUIConfiguration];
    if(configDictionary[@"canCancel"]) {
        NSString *canCancelValue = (NSString *)configDictionary[@"canCancel"];
        if ([canCancelValue isEqual: @"YES"]) {
            [config setCanCancel:true];
        } else {
            [config setCanCancel:false];
        }
        
    }
    if(configDictionary[@"logoImage"]) {
        [config setLogoImage:(UIImage *)configDictionary[@"logoImage"]];
    }
    if(configDictionary[@"backgroundColor"]) {
        [config setBackgroundColor:(UIColor *)configDictionary[@"backgroundColor"]];
    }
    
    [[AWSSignInManager sharedInstance] setDontFederate];
    
    AWSDDLogDebug(@"Presenting SignInViewController");
    dispatch_async(dispatch_get_main_queue(), ^{
        
        AWSSignInViewController *loginController;
        loginController = [AWSSignInViewController getAWSSignInViewControllerWithconfiguration: config];
        
        loginController.completionHandlerCustom = completionHandler;
        [loginController createInternalCompletionHandler];
        
        UINavigationController *navController = [[UINavigationController alloc] initWithRootViewController:loginController];
![IMG_2209](https://user-images.githubusercontent.com/5861783/66517100-80597200-eab0-11e9-8050-1a4409de09e8.jpeg)

// ADD THIS LINE 
        navController.modalPresentationStyle = UIModalPresentationFullScreen;
// ADD THIS LINE ABOVE

        [navigationController presentViewController:navController
                                           animated:YES
                                         completion:nil];
    });
}
```

**Comment by drochetti** [comment]
Hi @BillBunting, thank you for reporting to us. I appreciate the detailed report.

There was a change in iOS 13 indeed and I could repro it locally. We will work on it and follow up with you.

**Comment by BillBunting** [comment]
Fixed by 2.12.1. Thank you.
